Solved-Problems/Backspace_String_Compare/Main.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Stack;
```

Comment:
@rahulch-1 Please provide the problem-related details within your program file as mentioned within the rules and guidelines in [CONTRIBUTING.md](../../blob/main/CONTRIBUTING.md#instructions-and-guidelines).

```suggestion
// Problem Name: <problem-name>
// Approach:     <data-structure> (or) <algorithm>
// Diffculty:    <mentioned-diffculty>
// Link:         <link-to-the-problem>

import java.util.Stack;
```